content/en/news/releases/1.8.x/announcing-1.8.2/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Announcing Istio 1.8.2
+linktitle: 1.8.2
+subtitle: Patch Release
+description: Istio 1.8.2 patch release.
+publishdate: 2020-01-14
+release: 1.8.2
+aliases:
+    - /news/announcing-1.8.2
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.8.1 and Istio 1.8.2
+
+{{< relnote >}}
+
+## Changes
+
+- **Improved** `WorkloadEntry` auto-registration stability.
+  ([PR #29876](https://github.com/istio/istio/pull/29876))
+
+- **Improved** Multicluster install docs to include current
```

Comment:
hmm ... I actually wanted this backported to 1.8.0. I guess I didn't do it right?

Reply:
I moved them to the 1.8 release/upgrade notes